static/app/views/dashboardsV2/sortableWidget.tsx

```diff
@@ -1,31 +1,40 @@
-import {useEffect} from 'react';
-import * as React from 'react';
+import {ComponentProps, useEffect} from 'react';
 import {useSortable} from '@dnd-kit/sortable';
+import styled from '@emotion/styled';
 
+import {Organization} from 'sentry/types';
 import theme from 'sentry/utils/theme';
+import withOrganization from 'sentry/utils/withOrganization';
 
 import IssueWidgetCard from './issueWidgetCard';
 import {Widget, WidgetType} from './types';
-import WidgetCard from './widgetCard';
-import WidgetWrapper from './widgetWrapper';
-
-const initialStyles: React.ComponentProps<typeof WidgetWrapper>['animate'] = {
-  zIndex: 'auto',
-};
+import DiscoverWidgetCard from './widgetCard';
+import DnDKitWidgetWrapper from './widgetWrapper';
 
 type Props = {
   widget: Widget;
   dragId: string;
   isEditing: boolean;
+  hideDragHandle?: boolean;
```

Comment:
nit: can we move this to the end so optional inputs are at the bottom?

Reply:
100%, thanks for catching that!